.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,58 @@
+name: Generate release notes
+
+on:
+  workflow_dispatch:
+    inputs:
+      bc_commit_sha:
+        description: 'Commit SHA for the BC'
+        type: string
+        required: true
+      version:
+        description: 'Version to be released (for example, `9.2.1`)'
+        type: string
+        required: true
+
+env:
+  product: Elastic Agent
+  labels: docs,forwardport-main,release,skip-changelog,Team:Docs
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          ref: ${{ inputs.bc_commit_sha }}
+      - name: Generate release notes
+        run: |
```

Comment:
`nit`: i'm a big fan of using make for containing all the steps rused by the CI, it helps users to run the same as the CI without the need to copy and paste the snippet from the GH action